File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1AverageSpeed.java
package com.gmail.fattazzo.formula1world.domain;

/**
 * @author fattazzo
 *         <p/>
 *         date: 06/06/17
 */
public class F1AverageSpeed {

    public String units;

    public Double speed;

    public F1AverageSpeed(String units, Double speed) {
        this.units = units;
        this.speed = speed;
    }

    public F1AverageSpeed() {

    }

    @Override
    public String toString() {
        return "AverageSpeed{" +
                "units='" + units + '\'' +
                ", speed=" + speed +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1AverageSpeed.kt
package com.gmail.fattazzo.formula1world.domain

/**
 * @author fattazzo
 *
 *
 * date: 06/06/17
 */
class F1AverageSpeed {

    var units: String? = null

    var speed: Double? = null

    constructor(units: String, speed: Double?) {
        this.units = units
        this.speed = speed
    }

    constructor() {

    }

    override fun toString(): String {
        return "AverageSpeed{" +
                "units='" + units + '\'' +
                ", speed=" + speed +
                '}'
    }
}
